packages/rum-core/src/domain/view/viewMetrics/trackCumulativeLayoutShift.ts

```diff
@@ -138,5 +137,5 @@ function slidingSessionWindow() {
  * Check whether `layout-shift` is supported by the browser.
  */
 export function isLayoutShiftSupported() {
-  return supportPerformanceTimingEvent(RumPerformanceEntryType.LAYOUT_SHIFT)
+  return supportPerformanceTimingEvent(RumPerformanceEntryType.LAYOUT_SHIFT) && 'WeakRef' in window
```

Comment:
We used `WeakRef` to avoid memory leak when tracking CLS vital and assumed that browsers supporting layout-shift where also supporting WeakRef. Unfortunately that's not true and there is a few browsers versions that does support layout shift but not WeakRef. Unfortunately we were never testing on such browsers :( 

This bug affect the following browsers:
- Chrome 77 to 83
- Edge 79 to 83
- Opera 64 to 69

Reply:
Not sure how this is communicated, but it's worth mentioning in our documentation I guess

Reply:
I agree, although currently there is a lot of gaps on how we document the [features supported by browser](https://github.com/DataDog/browser-sdk/blob/main/packages/rum/BROWSER_SUPPORT.md)s and does not includes version.

My RFC around dropping IE11 proposes to remediates to this issue:
> Additionally we can update the feature support by browser to also include specific version numbers.